### PR TITLE
added validation on forgot password

### PR DESCRIPTION
**Pull request by Chee-Ng**

### CODE REVIEW

File: app/services/cognito/forgot_password.rb [hunk 1]
```diff
@@ -1,18 +1,24 @@
 module Cognito
   class ForgotPassword < BaseService
+    include ActiveModel::Validations
     attr_reader :email, :error
+    validates :email, format: { with: /\A([\w+\-].?)+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i }
```
Comment (iceraluk) [inline app/services/cognito/forgot_password.rb:5]: let's include some unit tests for this as well

File: db/schema.rb [hunk 2]
```diff
@@ -39,12 +39,12 @@
   end
 
   create_table "facilities_management_buildings", id: :uuid, default: nil, force: :cascade do |t|
+    t.text "user_email", null: false
```
Comment (iceraluk) [inline db/schema.rb:42]: we should not include any schema changes if no migrations have been added

Comment (Chee-Ng) [reply db/schema.rb:42]: I ran the migration from julian work, not sure why this is showing here tbh

File: app/services/cognito/forgot_password.rb (hunk 1 above)
Comment (Chee-Ng) [reply app/services/cognito/forgot_password.rb:5]: added

File: db/schema.rb (hunk 2 above)
Comment (iceraluk) [reply db/schema.rb:42]: @ManFromMons does these changes look right for the latest schema? or should we remove the schema changes from this PR

**Pull request closed without merging**